src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -39,6 +41,13 @@ public Command applyRequest(Supplier<SwerveRequest> requestSupplier) {
         return run(() -> this.setControl(requestSupplier.get()));
     }
 
+    public void optimizeCan() {
+        for (int i = 0; i > 3; i ++) {
```

Comment:
This for loop won't run, the second parameter needs to be flipped

Reply:
Good catch!